src/WebSockets.jl

```diff
@@ -57,6 +57,24 @@ end
 # Handshake
 
 
+function is_upgrade(req::HTTP.Request)
+    is_get = req.method == "GET"
+    # "upgrade" for Chrome and "keep-alive, upgrade" for Firefox.
+    is_upgrade = HTTP.hasheader(req, "Connection", "upgrade")
+    is_websockets = HTTP.hasheader(req, "Upgrade", "websocket")
+    return is_get && is_upgrade && is_websockets
+end
+
+
+function is_upgrade(res::HTTP.Response)
+    is_101 = res.status == 101
+    # "upgrade" for Chrome and "keep-alive, upgrade" for Firefox.
+    is_upgrade = HTTP.hasheader(res, "Connection", "upgrade")
+    is_websockets = HTTP.hasheader(res, "Upgrade", "websocket")
+    return is_101 && is_upgrade && is_websockets
+end
+
+
```

Comment:
Perhaps this could be expressed more succinctly something likes this:
```julia
is_websocket_upgrade(r::HTTP.Message) =
    (r isa Request && r.method == "GET" || r.status == 101) &&
    HTTP.hasheader(res, "Connection", "upgrade") &&
    HTTP.hasheader(res, "Upgrade", "webscoket")
```

Also, is any action needed per the  comment about "keep-alive, upgrade for Firefox" ?